org.eclipse.xtext.ide.tests/META-INF/MANIFEST.MF

```diff
@@ -17,6 +17,9 @@ Require-Bundle: org.eclipse.xtext;visibility:=reexport,
  org.eclipse.lsp4j;bundle-version="[0.19.0,0.20.0)";resolution:=optional,
  org.eclipse.lsp4j.jsonrpc;bundle-version="[0.19.0,0.20.0)";resolution:=optional,
  org.eclipse.xtext.testlanguages,
- org.eclipse.xtext.testlanguages.ide
+ org.eclipse.xtext.testlanguages.ide,
+ org.eclipse.emf.mwe2.lib;bundle-version="2.14.0",
+ org.aopalliance;bundle-version="1.0.0",
+ com.google.gson;bundle-version="2.9.1"
```

Comment:
It depends on how lsp4j has been bundled. Is it taken from Orbit? I've seen that before. For example, if you take mockito from Orbit you must remember to explicitly add its dependencies or it will compile but fail at run-time.